x-pack/plugins/reporting/public/management/report_listing_table.tsx

```diff
@@ -397,12 +397,12 @@ export class ReportListingTable extends Component<ListingPropsInternal, State> {
     return (
       <Fragment>
         {this.state.selectedJobs.length > 0 && (
-          <Fragment>
+          <div>
```

Comment:
if we replace the root Fragment with a FlexGroup with it's direction property set to `column` at the same time wrapping all children with a FlexItem, removing the FlexGroup that wraps the button's FlexItem, it works too.